tests/compliance_operator_test.go

```diff
@@ -45,9 +45,11 @@ const (
 	unusedProfile = "rhcos4-anssi-bp28-high"
 )
 
-const defaultWaitTime = 10 * time.Second
+const defaultWaitTime = 30 * time.Second
+const defaultSleepTime = 5 * time.Second
+const defaultTickTime = 2 * time.Second
 
-func getCurrentComplianceResults(t *testing.T) (rhcos, ocp *storage.ComplianceRunResults) {
+func getCurrentComplianceResults(t *testing.T, a assert.TestingT, r require.TestingT) (rhcos, ocp *storage.ComplianceRunResults) {
```

Comment:
Why do we need t, a, r if all are `*testing.T`?
```suggestion
func getCurrentComplianceResults(t *testing.T) (rhcos, ocp *storage.ComplianceRunResults) {
```

Reply:
I started passing a custom interface like:

```golang
type T interface {
	Errorf(format string, args ...interface{})
	Fatalf(format string, args ...interface{})
	FailNow()
}
```
and then discovered that we already have one in `pkg/testutils` 🤦 

It should be much cleaner now.